lib/capybara/webkit/matchers.rb

```diff
@@ -7,11 +7,11 @@ module RspecMatchers
           actual.error_messages.any?
         end
 
-        failure_message_for_should do |actual|
+        failure_message do |actual|
```

Comment:
Great point. Upon further inspection of the example I saw previously, it's a bit more complicated than a simple `alias_method`:

```ruby
Spec::Matchers.constants.each do |c|
  if Class === (klass = Spec::Matchers.const_get(c))
    if klass.public_instance_methods.any? {|m| ['failure_message_for_should',:failure_message_for_should].include?(m)}
      klass.class_eval do
        alias_method :failure_message, :failure_message_for_should
      end
    end
    if klass.public_instance_methods.any? {|m| ['failure_message_for_should_not',:failure_message_for_should_not].include?(m)}
      klass.class_eval do
        alias_method :negative_failure_message, :failure_message_for_should_not
      end
    end
  end
end
```